CatNote.API/appsettings.json

```diff
@@ -8,5 +8,11 @@
   "AllowedHosts": "*",
   "ConnectionStrings": {
     "DefaultConnection": "Server=localhost;Database=CatNote;Trusted_Connection=True;Trust Server Certificate=True;"
+  },
+  "Auth0": {
+    "Domain": "dev-ujxvenaomkidb36h.us.auth0.com",
+    "Audience": "https://localhost:7165/api",
+    "ClientSecret": "",
```

Comment:
Why is this empty?

Reply:
Keys and similar items should be stored out of sight and get when needed(example for deployment)